doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-enabling-security-management-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='business-central-enabling-security-management-proc']
+
+= Enabling the security management feature
```

Comment:
@RishiRajAnand The security management feature is enabled by default, isn't it? Do we need those steps in the documentation for the deployed Business Central?

Reply:
@kaldesai Could you please remove this file as it is not used anymore?

Reply:
@tomasdavidorg, I have removed this file.